include/wx/menuitem.h

```diff
@@ -122,6 +122,13 @@ class WXDLLIMPEXP_CORE wxMenuItemBase : public wxObject
     // set the accel for this item - this may also be done indirectly with
     // SetText()
     virtual void SetAccel(wxAcceleratorEntry *accel);
+
+    // add the accel to extra accels list
+    void AddExtraAccel(wxAcceleratorEntry *accel);
+
+    wxVector<wxString> GetExtraAccels() const { return m_extra_accels; }
```

Comment:
Doesn't matter much for vectors of this size, but it should still arguably be returned by const reference rather than copied.